testInstall.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
```

Comment:
⚠️  couldn't really determine if this is used or not (is it just a manual test for our own purposes?)... kept it, but not sure if its needed

Reply:
yeah, I believe so cc @chickenbug. maybe you guys can sync up to utilize this script pre-release to ensure everything is working (I just don't recall exactly how it works)

Reply:
Whoa what a throwback. Yeah I _think_ this tests the install.js script by running npm in a temp subfolder. I don't fully remember though

Reply:
ok ran this locally, looks like we are `npm install`ing to a temp directory our package (`mongodb-realm-cli`) and then ensuring the `realm-cli` and `transpiler` executables are present.

going to leave this in since it works as a manual/local check.  if/when we can get this release process more automated (ideally through evergreen, we won't need an team member's `npm` access ((nor good internet)) to actually push new releases), we can use this script as a post-check to make sure everything ran smoothly

alternatively, we could also include this as a check on "non-patchable" commits just to make sure we're still publicly available.  will file these ideas as a follow up ticket to pursue in the v2 release of this

Reply:
https://jira.mongodb.org/browse/REALMC-8411